web/pages/api/dao/create.ts

```diff
@@ -17,11 +17,14 @@ async function postHandler(
   res: NextApiResponse<Dao | string>
 ) {
   try {
+    console.log('Got request to create DAO', req.body);
     const prisma = new PrismaClient();
     const result = await prisma.dao.create({
       data: {
         publicKey: req.body.publicKey,
         title: req.body.title,
+        about: req.body.about,
+        amount: parseFloat(req.amount) * 100000000, // convert to sats
```

Comment:
REQ: would use sats everywhere in backend/api to make it easy and not care about the notation. FE takes care of transforming user input in sats.

Reply:
Yes - everything in back-end is sats, but this conversion is on the back-end and comes in in BTC from the front-end so we need to convert it